democracy/models/comment.py

```diff
@@ -11,6 +11,7 @@ class BaseComment(BaseModel):
     parent_model = None  # Required for factories and API
     title = models.CharField(verbose_name=_('title'), blank=True, max_length=255)
     content = models.TextField(verbose_name=_('content'))
+    authorization_code = models.CharField(verbose_name=_('authorization code'),  max_length=255, blank=True, null=True)
```

Comment:
CharFields generally shouldn't be nullable, since then you'll have two falsy values (`NULL` and `""`).

Is the field really required to be 255 characters long?